Source/WebKit/GPUProcess/graphics/WebGPU/WebGPUObjectHeap.cpp

```diff
@@ -53,10 +53,119 @@
 #include "RemoteTexture.h"
 #include "RemoteTextureView.h"
 
+#if PLATFORM(COCOA)
+#include <notify.h>
+#include <unordered_map>
+#endif
+
 namespace WebKit::WebGPU {
 
+#if PLATFORM(COCOA)
```

Comment:
There's nothing platform specific about printing a heap; can we keep these outside of the guard, and just guard the `dispatch` stuff that really is platform specific?